src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -148,6 +150,7 @@ class Client extends BaseTransport implements QueryTransport, WritingInterface,
      */
     private $coreNamespaces = array(
         NamespaceRegistryInterface::PREFIX_EMPTY => NamespaceRegistryInterface::NAMESPACE_EMPTY,
+        NamespaceRegistry::PREFIX_REP => NamespaceRegistry::NAMESPACE_REP,
```

Comment:
@dbu Is this solution ok? Have now moved the constant to jackalope/jackalope instead of phpcr/phpcr.